packages/@webex/plugin-meetings/test/unit/spec/meetings/index.js

```diff
@@ -1287,7 +1293,8 @@ describe('plugin-meetings', () => {
             const meeting = await webex.meetings.createMeeting('test destination', 'test type');
 
             const expectedMeetingData = {
-              permissionToken: 'PT',
+              permissionToken:
+                'eyJhbGciOiJIUzI1NiJ9.eyJleHAiOiIxMjM0NTYiLCJwZXJtaXNzaW9uIjp7InVzZXJQb2xpY2llcyI6eyJhIjp0cnVlfX19.wkTk0Hp8sUlq2wi2nP4-Ym4Xb7aEUHzyXA1kzk6f0V0',
```

Comment:
Why do we want to put a token like this instead of PT or any test word?

This question goes to all the places where this change is made

Reply:
Same as above.